yarn.lock

```diff
@@ -9,6 +9,13 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/code-frame@^7.10.4":
```

Comment:
It's not clear to me why this changed so much -- the tree of dependencies is the same aside from `eslint-config-tsdx`, so there shouldn't be that much change